### PR TITLE
Cleanup of 2-gofmt-and-friends

### PR DESCRIPTION
**Pull request by colezlaw**
A couple of grammar and command updates to 2-gofmt-and-friends.

### CODE REVIEW

**Comment by colezlaw** [comment]
I signed it!

File: 1-source-code/2-writing/2-gofmt-and-friends.md [hunk 1]
```diff
@@ -113,31 +113,31 @@ Cool, right?
...
 import statements.
 
-`goimports` will remove import statements for any packages that are not used in the code
-below. Even cooler, `goimports` will also find references to packages that are not imported
+`goimports` will remove import statements for any packages that are not used in the code.
+Even cooler, `goimports` will also find references to packages that are not imported
 and add them.
 
-How does this magic work, you say? It's quite straight forward, given an expression such as
+You might ask how this magic works. It's quite straightforward. Given an expression such as
```
Comment (mrxinu) [inline 1-source-code/2-writing/2-gofmt-and-friends.md]: Rather than changing @campoy's voice here I would just change the word "say" to "ask".

Comment (colezlaw) [inline 1-source-code/2-writing/2-gofmt-and-friends.md]: Yeah - let's do that. The grammarian in me couldn't figure out how or where to put the quotes in it, though. A technical writer I know often says "if you can't figure out how to say it, say something different" which has served me well, except when editing other peoples' work.

Comment (mrxinu) [inline 1-source-code/2-writing/2-gofmt-and-friends.md]: That's great advice! I'll have to remember that one.

**Comment by campoy** [comment]
Thanks, @colezlaw!

**Pull request merged**